### PR TITLE
new karate whoop tune late 2024

### PR DESCRIPTION
**Pull request by sugaarK**
late 2024 light weight whoop race tune. developed with real black boxing thanks to the Betafpv 5n1. tested on NBD hummingbird v3 RS and Happy Model MOB 65 HDZ aio

### CODE REVIEW

**Comment by limonspb** [comment]
is this intended to be for 4.4, not 4.5? 0_o

**Comment by sugaarK** [comment]
> is this intended to be for 4.4, not 4.5? 0_o

ohh yeah sorry fixed

File: presets/4.5/tune/karate/karate_whoop_late_2024.txt [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+#$ DESCRIPTION:
+#$ DESCRIPTION: ## Second note... Radio links:
+#$ DESCRIPTION: 1. Make sure your radio firmware is up to date using either EdgeTX or OpenTX
+#$ DESCRIPTION: 1. Make sure your **ADC Filter is OFF** in the hardware page
+#$ DESCRIPTION: 1. Go to the radio (RC_LINK) presets and apply the correct setup for your system and link speed
+
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/490
+
+#$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
+#$ INCLUDE: presets/4.4/tune/defaults.txt
```
Comment (limonspb) [inline presets/4.5/tune/karate/karate_whoop_late_2024.txt:39]: here also 4.4

ANd need to include filters defaults too i guess

File: presets/4.5/tune/karate/karate_whoop_late_2024.txt [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    set dyn_idle_p_gain = 35
+    set dyn_idle_start_increase = 60
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Vbat sag compensation ( recommended )
+    set vbat_sag_compensation = 100
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): Over clock f411 with ICM42688p only
+        set cpu_overclock = 120MHZ
```
Comment (limonspb) [inline presets/4.5/tune/karate/karate_whoop_late_2024.txt:81]: extra spaces

File: presets/4.5/tune/karate/karate_whoop_late_2024.txt [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+set thrust_linear = 20
+set vbat_max_cell_voltage = 445
+
+#$ OPTION BEGIN (CHECKED): Dynamic idle for sub 20g whoop ( recommended )
+    set dyn_idle_min_rpm = 150
+    set dyn_idle_p_gain = 35
+    set dyn_idle_start_increase = 60
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Vbat sag compensation ( recommended )
```
Comment (limonspb) [inline presets/4.5/tune/karate/karate_whoop_late_2024.txt:76]: intended to be checked? I don't remember if sag compensation is for throttle, or PIDs adjsutments?

Comment (sugaarK) [reply presets/4.5/tune/karate/karate_whoop_late_2024.txt:76]: pids and yeah you should use it

File: presets/4.5/tune/karate/karate_whoop_late_2024.txt [hunk 4]
```diff
@@ -0,0 +1,183 @@
+#$ TITLE: Karate whoop late 2024
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: TUNE
+#$ STATUS:  EXPERIMENTAL
```
Comment (limonspb) [inline presets/4.5/tune/karate/karate_whoop_late_2024.txt:4]: extra space. This one is important when you search for all experimental presets to turn them community

**Comment by sugaarK** [comment]
good catch's I lowered the dynamic a bit more too as I been flying it all afternoon and feel its a bit better

**Review by limonspb — APPROVED** [review]

File: presets/4.5/tune/karate/karate_whoop_late_2024.txt [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+    #$ OPTION END
+    
+    #$ OPTION BEGIN (UNCHECKED): throttle expo for 45% hover throttle
+        set thr_mid = 45
+        set thr_expo = 65
+    #$ OPTION END
+    
+
+#$ OPTION_GROUP END
+
```
Comment (haslinghuis) [inline presets/4.5/tune/karate/karate_whoop_late_2024.txt:184]: :space_invader: 
```suggestion

#$ OPTION_GROUP END
```

Comment (sugaarK) [reply presets/4.5/tune/karate/karate_whoop_late_2024.txt]: fixed

**Review by haslinghuis — APPROVED** [review]

**Comment by Sb-Art11** [comment]
Will it work with the Air65?

**Comment by haslinghuis** [comment]
Yes

**Comment by haslinghuis** [comment]
Yes - give it a try

**Comment by sugaarK** [comment]
> Will it work with the Air65?

was made with the bfpv 5n1 aio form the air65... it should work pretty good

**Pull request merged**